src/image_resizer/util.clj

```diff
@@ -7,7 +7,7 @@
 (defn buffered-image [image]
   (if (instance? BufferedImage image)
     image
-    (ImageIO/read image)))
+    (ImageIO/read ^java.io.ByteArrayInputStream image)))
```

Comment:
this cast is failing if image is a file from disk (travis is complaining)

will fix and update